src/main/java/mate/academy/bookstore/service/BookService.java

```diff
@@ -14,4 +15,6 @@ public interface BookService {
     BookDto update(Long id, CreateBookRequestDto bookRequestDto);
 
     void deleteById(Long id);
+
+    List<BookDto> searchBooks(BookSearchParametersDto searchParameters);
```

Comment:
guess **search** for method name will be enough, as we are already in the `BookService`